tools/tests/form-control/xhr.test.js

```diff
@@ -0,0 +1,54 @@
+const template = `<div class="form-wrapper">
+<gameface-form-control id="xhr-form" action="http://localhost:${SERVER_PORT}/user" method="get">

```

Comment:
SERVER_PORT is defined in the forms test and it is global, so we can access it here. Is it possible to load this test before the form-control test?